packages/icon/src/scss/_build/_icons-emit.scss

```diff
@@ -107,6 +107,9 @@
 @include icon('how-youll-learn--performance-presentation-graph');
 @include icon('how-youll-learn--seo-search-page');
 @include icon('how-youll-learn--users');
+@include icon('interface-essential--check-circle-1');
```

Comment:
You shouldn't need to edit the build files if you follow the icon package maintenance process. Also, I believe the reason why these new icons look different is that Jen or Laura actually modified the stroke widths for Future Students so the same needs to be done for these as well. Using the proper build process will also give you updated documentation.

Reply:
They've included their export process here: https://www.figma.com/file/vG3eumt1NqHAIe0AQjkuCX/Design-system-WIP?node-id=232%3A0